.github/workflows/helm-chart-tests.yaml

```
@@ -0,0 +1,733 @@
+        run: |
+          poetry install
+
+      - name: ${{ matrix.instances.name }}
+        if: (matrix.instances.if == 'basic' && (github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && inputs.test_type == 'all') || github.ref == 'refs/heads/kubeblocks' || (github.event_name == 'pull_request' && github.head_ref == 'kubeblocks'))) || (matrix.instances.if == 'full' && (env.TEST_MODE == 'full' || github.ref == 'refs/heads/kubeblocks' || (github.event_name == 'pull_request' && github.head_ref == 'kubeblocks')))
+        env:
+          SERVICE_ID: ${{ vars.OMNISTRATE_INTERNAL_SERVICE_ID }}
+          ENVIRONMENT_ID: ${{ vars.OMNISTRATE_INTERNAL_DEV_ENVIRONMENT }}
+          SUBSCRIPTION_ID: sub-GJPV3NoNC0
+          OMNISTRATE_USERNAME: ${{ secrets.OMNISTRATE_USERNAME }}
+          OMNISTRATE_PASSWORD: ${{ secrets.OMNISTRATE_PASSWORD }}
+        run: |
```

actionlint (1.7.8)

property "tls" is not defined in object type {cloudprovider: string; cloudregion: string; clusterreplicas: number; hostcount: number; if: string; instancetype: string; maxmemory: string; name: string; newinstancetype: string; resourcekey: string; testfile: string; tests: string; tiername: string}

(expression)